docs/en/reference/index.rst

```diff
@@ -8,8 +8,8 @@ exceptions/differences/extensions.
 
 - Keep the nesting of control structures per method as small as possible
 - Abstract exception class names and exception interface names should be suffixed with ``Exception``
-- Abstract classes should not be prefixed with ``Abstract``
-- Interfaces should not be suffixed with ``Interface``
+- Abstract classes should not be suffixed or prefixed with ``Abstract``
+- Interfaces should not be suffixed or prefixed with ``Interface``
 - Concrete exception class names should not be suffixed with ``Exception``
```

Comment:
Why not add prefix to exception as well?

Reply:
Note that no Exception prefix is currently not enforced by any current sniff.